contracts/Market.sol

```diff
@@ -230,6 +241,27 @@ contract Market is IMarket, Ownable, ERC721 {
 		return Math.max(wager, (trueOdds * wager) / OddsLib.PRECISION);
 	}
 
+	function multiBack(
+		BackData[] calldata backDataList
+	) external returns (uint256[] memory) {
+		uint256[] memory indexes = new uint256[](backDataList.length);
```

Comment:
do we need a new array? can't you reference the back data with `BackData[i]` in the loop?